ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/views/NoActiveUserManagementView.kt

```diff
@@ -56,31 +66,53 @@ private fun ContentUnavailableView(
     description: String?,
     modifier: Modifier = Modifier,
 ) {
-    Column(
-        modifier = modifier
-            .fillMaxWidth()
-            .padding(ContentUnavailableViewPadding),
-        horizontalAlignment = Alignment.CenterHorizontally,
+    Surface(
+        modifier = modifier,
+        shape = RoundedCornerShape(CustomerCenterConstants.Card.ROUNDED_CORNER_SIZE),
+        color = MaterialTheme.colorScheme.surface,
     ) {
-        Icon(
-            imageVector = Icons.Rounded.Warning,
-            contentDescription = null,
-            modifier = Modifier.size(ContentUnavailableIconSize),
-        )
-
-        Text(
-            text = title,
-            style = MaterialTheme.typography.headlineSmall,
-            modifier = Modifier.padding(top = ContentUnavailableViewPaddingTopTitle),
-        )
+        Column(
+            modifier = Modifier.fillMaxWidth()
+                .padding(all = 20.dp),
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+            Icon(
+                imageVector = Icons.Rounded.Info,
+                contentDescription = null,
+                modifier = Modifier.size(ContentUnavailableIconSize),
+            )
 
-        description?.let {
             Text(
-                text = it,
-                style = MaterialTheme.typography.bodyLarge,
-                textAlign = TextAlign.Center,
-                modifier = Modifier.padding(top = ContentUnavailableViewPaddingTopDescription),
+                text = title,
+                style = MaterialTheme.typography.titleMedium,
+                modifier = Modifier.padding(vertical = ContentUnavailableViewPaddingTitle),
             )
+
+            description?.let {
+                Text(
```

Comment:
Small thing but should we add a bottom padding here of `ContentUnavailableViewPaddingTitle`? Mostly thinking about having the bottom padding  here be the same whether there is a description or not.

Reply:
049e317cac847d228e53e47f372bdd4055b7e47b